.github/workflows/package-workflow.yml

```diff
@@ -0,0 +1,53 @@
+name: E4E Pipeline
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    env:
+      PROJECT_NAME: explorer-for-endevor
+
+    strategy:
+      # Continue to run tests on the other systems if one fails
+      fail-fast: false
+      matrix:
+        node-version: [12.x, 14.x, 16.x]
+        os: [windows-latest, ubuntu-latest, macos-latest]
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: yarn install
+
+      - name: Audit
+        id: audit
+        continue-on-error: true
+        run: yarn audit
+
+      - name: Lint
+        run: yarn workspace explorer-for-endevor lint
```

Comment:
Hmm, it seems like you can specify just: `yarn lint`, which will run the script for all the existing workspaces, because now, you are testing e4e package only, but we also have endevor, vscode-wrapper, profiles and etc packages there as well.

The same stuff for test and build steps